src/observability/input-summary.json

```diff
@@ -0,0 +1,430 @@
+{
+    "Positive test 1": [
```

Comment:
I don't think the notion of postive/negative cases applies here. The function which is under test, doesn't return a positive/negative value. Its just return some value. Comparing the output of the function with a wrong value doesn't make sense.

Reply:
The reason for adding the same name on both the input and output array was to ensure that the comparison happens on the correct input/output array. Also positive and negative were just added as keywords to search through and redirect to respective test case functions. Should we change that and keep only one test case?